src/app/dashboard-widgets/add-codebase-widget/codebase-item/codebase-item.component.ts

```diff
@@ -1,15 +1,37 @@
 import {
   Component,
-  Input
+  Input,
+  OnInit
 } from '@angular/core';
 
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators/map';
+
 import { Codebase } from '../../../space/create/codebases/services/codebase';
+import { GitHubRepoDetails } from '../../../space/create/codebases/services/github';
+import { GitHubService } from '../../../space/create/codebases/services/github.service';
 
 @Component({
   selector: 'fabric8-add-codebase-widget-codebase-item',
   templateUrl: './codebase-item.component.html',
-  styleUrls: ['./codebase-item.component.less']
+  styleUrls: ['./codebase-item.component.less'],
+  providers: [GitHubService]
 })
-export class CodebaseItemComponent {
+export class CodebaseItemComponent implements OnInit {
+
   @Input() codebase: Codebase;
+  lastUpdated$: Observable<string>;
```

Comment:
What's the `$` convention? I'm not familiar with it.

Reply:
It's used to signify that the property is an Observable. I'm not opposed to chopping it off.